lms/templates/bookmarks/bookmarks_results.underscore

```diff
@@ -0,0 +1,58 @@
+<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
+<h2 class="bookmarks-results-header"><%= gettext("My Bookmarks") %></h2>
+
+<% if (bookmarks.length) { %>
+
+    <div class='bookmarks-results-list'>
+        <% _.each(bookmarks, function(bookmark, index) { %>
+             <a class="bookmarks-results-list-item" href="<%= bookmark.get('usage_id') %>" aria-describedby="bookmark-type-<%= index %> bookmark-date-<%= index %>" target="_blank">
+                <div class="list-item-content">
+                    <div class="list-item-left-section">
+                        <h3> <%= bookmark.get('breadcrumbTrail') %> </h3>
+                        <p id="bookmark-date-<%= index %>" class="list-item-date"> <%= gettext("Bookmarked on") %> <%= bookmark.get('userFriendlyDate') %> </p>
+                    </div>
+
+                    <p id="bookmark-type-<%= index %>" class="list-item-right-section">
+                        <span aria-hidden="true">View</span>
+                        <i class="icon fa fa-arrow-right" aria-hidden="true"></i>
+                    </p>
+                </div>
+            </a>
+        <% }); %>
+
+        <!--Below anchor tag is temporarily placed here just to test accessibility-->
+        <a class="bookmarks-results-list-item" href="http://www.edx.org" aria-labelledby="bookmark-link-3" aria-describedby="bookmark-type-3 bookmark-date-3" target="_blank">
+            <div class="list-item-content">
+                <div class="list-item-left-section">
+                    <h3 id="bookmark-link-3"> FIRST > SECOND > THIRD </h3>
+                    <p id="bookmark-date-3" class="list-item-date"> Bookmarked on January 1, 2050 </p>
+                </div>
+
+                <p id="bookmark-type-3" class="list-item-right-section">
+                    <span aria-hidden="true">View</span>
+                    <i class="icon fa fa-arrow-right" aria-hidden="true"></i>
+                </p>
+            </div>
+        </a>
+
+    </div>
+
+<% } else { %>
+
+    <div class="bookmarks-empty" tabindex="0">
+        <div class="bookmarks-empty-header">
+            <i class="icon fa fa-bookmark-o bookmarks-empty-header-icon" aria-hidden="true"></i>
+            It looks like your bookmarks list is empty!
+            <br>
+        </div>
+        <div  class="bookmarks-empty-detail">
+            <span class="bookmarks-empty-detail-title">
+                <%= gettext("Use bookmarks to quickly reference courseware pages you would like to return to. Toggle the bookmark button on courseware page to save that page to this list. An example bookmark toggle is shown here for reference.") %>
```

Comment:
Use bookmarks to help you easily return to courseware pages. To bookmark a page, select **Bookmark** in the upper right corner of that page. To see a list of all your bookmarks, select **Bookmarks** in the upper left corner of any courseware page.